apollo-router/tests/snapshots/integration_tests__traced_basic_composition.snap

```diff
@@ -104,211 +104,6 @@ expression: get_spans()
         }
       },
       "children": {
```

Comment:
could we change the test log level?
I think moving those to TRACE is a valid point, however I think we want to keep snapshotting these spans so we can track regressions.

Reply:
Sure. I've set the basic_composition and basic_request to TRACE.
done: 220ccaa87799d1a010d3267948e4783a80382f43